internal/bminventory/inventory_v2_handlers.go

```diff
@@ -70,6 +70,13 @@ func (b *bareMetalInventory) V2DeregisterCluster(ctx context.Context, params ins
 		return common.NewApiError(http.StatusNotFound, err)
 	}
 
+	if swag.StringValue(cluster.Status) == models.ClusterStatusInstalling {
+		return common.NewApiError(
+			http.StatusConflict,
+			fmt.Errorf("cluster %s can not be removed while being installed", cluster.ID),
```

Comment:
Will the user see this error in the UI? Can we help them understand what they can do to overcome this error? (Maybe adding something along the lines of "Please wait for the installation to timeout or reset the installation")

Reply:
I think this action doesn't exist in the UI, but I agree that error messages are better when they include how to fix the thing.

Reply:
Done.